manifests/config.pp

```diff
@@ -5,7 +5,10 @@
   }
 
   concat {
-    '/etc/postgrey/whitelist_clients':
+   '/etc/postgrey/whitelist_clients':
+     require => [
```

Comment:
This shouldn't be needed, as the ::config class already depends on the ::install class

Reply:
I don't see ::config depending on ::install, unless if you mean through the ordering in init.pp? If you were just to include ::config that would not work.

I also assume you're saying that based on the assumption that the package creates /etc/postgrey, which isn't true for centos/fedora (see below), so it does seem to be necessary at least on those systems.